WooCommerce/WooCommerceTests/Mocks/MockPaymentCaptureOrchestrator.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+@testable import WooCommerce
+import Yosemite
+
+final class MockPaymentCaptureOrchestrator: PaymentCaptureOrchestrating {
+    var mockCollectPaymentHandler: ((_ onPreparingReader: () -> Void,
+                                     _ onWaitingForInput: (Yosemite.CardReaderInput) -> Void,
+                                     _ onProcessingMessage: () -> Void,
+                                     _ onDisplayMessage: (String) -> Void,
+                                     _ onProcessingCompletion: (Yosemite.PaymentIntent) -> Void,
+                                     _ onCompletion: (Result<WooCommerce.CardPresentCapturedPaymentData, Error>) -> Void) -> Void)? = nil
```

Comment:
We don't seem to need WooCommerce here, by importing it as `@testable` finds it correctly: 

```suggestion
                                     _ onCompletion: (Result<CardPresentCapturedPaymentData, Error>) -> Void) -> Void)? = nil
```

Reply:
Yeah, this is from adding protocol conformances. It's really annoying, I thought I'd caught all of them.